_states/keystone_policy.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+# Copyright 2017 Mirantis, Inc.
```

Comment:
It's better to use `Mirantis, Inc. et al.` to include other contributors. Or don't add the license at all as it's already in root of this repository.